infrastructure/variables.tf

```diff
@@ -5,7 +5,7 @@ variable "product" {
 
 variable "microservice" {
   type = "string"
-  default = "api"
+  default = "service"
```

Comment:
This doesn't look right to me micro service as service ?I would expect some thing similar to 
`"service-auth-provider` or 'pdf-service`